pom.xml

```diff
@@ -64,6 +64,7 @@
         <version.carlspring.npm.metadata>1.0-SNAPSHOT</version.carlspring.npm.metadata>
 
         <version.apache.commons-io>2.6</version.apache.commons-io>
+        <version.apache.commons-compress>1.19</version.apache.commons-compress>
```

Comment:
I just noticed this...
Would you mind changing all the version properties that have hyphens in them to dots? (Here and in the `strongbox-proto-parent`, as well as all the other strongbox projects where they are used?

Reply:
What is the reason to do this? :)

Reply:
It's cosmetic and for consistency only... All the other version properties are using dots only and not a mixture of dots and hyphens.

Reply:
Okay then, I will fix it todays evening

Reply:
Great! Thanks a lot! :)

Reply:
@carlspring This PR has been updated.
For `strongbox-proto-parent` fix is in PR: https://github.com/strongbox/strongbox-proto-parent/pull/7

Reply:
Fantastic!

...And you've checked that the `strongbox` project doesn't reference any of these properties, right? (The dependencies should be coming via the `dependencyManagement` section and be versionless across all the projects that extend these parents, but you never know...

Reply:
Yes, I have checked that :)

Reply:
Excellent! Thank you! :)